### DISCUSSION TITLE
Highlight basic blocks in linear disassembly view

### DISCUSSION THREAD

**Discussion by sab4tg** [Vector35/binaryninja-api#2005] [General] [answered]
Hello - I'd like to be able to visually draw attention to basic blocks of code by highlighting them various colors. When I use the function ```basic_block.set_user_highlight(color)``` this highlights the specified block in the disassembly graph view, but not in the linear disassembly view; ideally I'd like to do both. Is there a way to achieve this using ```set_user_highlight``` or is a different function required in this case?

Thanks!

**Comment by psifertex** [comment, marked as answer]
No such API exists at this time unfortunately. You can track #1940 if you'd like to know when the feature is added.
